config/secrets.yml.example

```diff
@@ -12,11 +12,17 @@
 
 development:
   secret_key_base:
+  PHONE_TOKEN_SECRET:
+  PHONE_OAUTH_APP_ID:
 
 test:
   secret_key_base:
+  PHONE_TOKEN_SECRET:
+  PHONE_OAUTH_APP_ID:
 
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  PHONE_TOKEN_SECRET: <%= ENV["PHONE_TOKEN_SECRET"] %>
+  PHONE_OAUTH_APP_ID: <%= ENV["PHONE_OAUTH_APP_ID"] %>
```

Comment:
Wouldn't this be something that is fixed?

Reply:
Someone would have to create an OAuth application that the token will formally be issued to, and it only makes sense to have that ID configurable externally. And the secret auth token should definitely not be static in the code.